docs/faq.rst

```diff
@@ -43,6 +43,21 @@ navigating to your package root directory and typing :code:`pre-commit run --all
 on how to run the tools from the command line.
 
 
+**Q:** Pre-commit fails with the following error:
+
+..code-block:: bash
+
+    An error has occurred: InvalidManifestError:
+    ==> File /users/jappel/.cache/pre-commit/repozhhauwxe/.pre-commit-hooks.yaml
```

Comment:
Maybe make it more general:
`/users/<username>/.cache/pre-commit/repozhhauwxe/.pre-commit-hooks.yaml`